.changeset/silent-cougars-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/util': minor
+---
+
+[Compatibility Note] Stop using `VCAP_SERVICES` to determine the log format. Use `NODE_ENV=production` or `cds.env.features.kibana_formatter=true` for `kibana` format. Otherwise, `local` format will be used.
```

Comment:
I think as long as `NOVE_ENV` is set to "production", which is set by `nodejs_buildpack` on CF by default, `kibana` will be used the same as before since `VCAP_SERVICES` is always set on CF as well.

Reply:
This is also my understanding. You never know though for specific cases. With the compatiblity note I think it should be ok 😊